data/etc/foglamp.json

```diff
@@ -0,0 +1,4 @@
+{ "default storage plugin" : "postgres",
```

Comment:
If we are going to have this here then we should update the storage service to take this into account. It already has its own notion of what is default built into the code. I don't like duplicating this. I suggest raise a Jira that this has caused duplication and we fix it in a future sprint.